Octokit.Tests.Integration/fixtures/OrganizationsHooksCollection.cs

```diff
@@ -0,0 +1,10 @@
+﻿using Xunit;
+
+namespace Octokit.Tests.Integration.fixtures
```

Comment:
I'll undo the changes in the 2 classes and make it Fixtures in the classes I am adding here